grpc-haskell.cabal

```diff
@@ -19,7 +19,7 @@ Flag Debug
 flag with-examples
   description: Also build example executables.
   manual:      True
-  default:     False
+  default:     True
```

Comment:
It would be nice to provide the command line invocation to build with examples in the Tutorial so others don't have to discover it too.